ReactiveCocoaTests/Swift/PropertySpec.swift

```diff
@@ -14,66 +14,258 @@ import ReactiveCocoa
 			}
 
-			pending("should yield a producer that sends the current value then all changes") {
+			it("should yield a producer that sends the current value then all changes") {
+				let initialValue = "StringValue"
+				let subsequentValue = "NewStringValue"
+
+				let mutableProperty = MutableProperty(initialValue)
+
+				var valueSent: String?
+				var signalCompleted = false
+
+				mutableProperty.producer.start(next: { value in
+					valueSent = value
+				},
+				completed: {
+					signalCompleted = true
+				})
+
+				expect(valueSent).to(equal(initialValue))
+				expect(signalCompleted).to(beFalsy())
+
+				mutableProperty.value = subsequentValue
+
+				expect(valueSent).to(equal(subsequentValue))
+				expect(signalCompleted).to(beFalsy())
 			}
 
-			pending("should complete its producer when deallocated") {
+			it("should complete its producer when deallocated") {
+				let propertyValue = "StringValue"
+				var mutableProperty: MutableProperty? = MutableProperty(propertyValue)
+
+				var signalCompleted = false
+
+				mutableProperty?.producer.start(next: { value in
+				},
+				completed: {
+					signalCompleted = true
+				})
+
+				mutableProperty = nil
+				expect(signalCompleted).to(beTruthy())
 			}
 		}
 
 		describe("PropertyOf") {
-			pending("should pass through behaviors of the input property") {
+			it("should pass through behaviors of the input property") {
+				let propertyValue = "StringValue"
+
+				let constantProperty = ConstantProperty(propertyValue)
+				let propertyOf = PropertyOf(constantProperty)
+
+				var valueSent: String?
+
+				propertyOf.producer.start(next: { value in
+					valueSent = value
+				})
+
+				expect(valueSent).to(equal(propertyValue))
 			}
 		}
 
 		describe("binding") {
 			describe("from a Signal") {
-				pending("should update the property with values sent from the signal") {
+				it("should update the property with values sent from the signal") {
+					let finalPropertyValue = 815
+					var signalObserver: SinkOf<Event<Int, NoError>>?
+
+					let sendPropertyUpdate = {
+						sendNext(signalObserver!, finalPropertyValue)
+					}
+
+					let signal = Signal<Int, NoError>({ observer in
+						signalObserver = observer
+
+						return SimpleDisposable()
+					})
+
+					let mutableProperty = MutableProperty<Int>(0)
+
+					mutableProperty <~ signal
+
+					sendPropertyUpdate()
+					expect(mutableProperty.value).to(equal(finalPropertyValue))
 				}
 
-				pending("should tear down the binding when disposed") {
+				it("should tear down the binding when disposed") {
+					let signalDisposable = SimpleDisposable()
+
+					let signal = Signal<Int, NoError>({ observer in
+						return signalDisposable
+					})
+
+					let mutableProperty = MutableProperty<Int>(0)
+
+					let bindingDisposable = mutableProperty <~ signal
+					bindingDisposable.dispose()
+
+					// This test is failing: disposing the binding isn't disposing the signal, or am I doing something wrong?
```

Comment:
@jspahrsummers do you see anything obviously wrong with this test?
Am I misinterpreting "tear down the binding"?